0.14.5: ADIOS1 Link Error on macOS

There is likely an `export` missing for these symbols:
```
  >> 115    Undefined symbols for architecture x86_64:
     116      "openPMD::internal::operationAsString(openPMD::Operation)", refer
            enced from:
     117          openPMD::ParallelADIOS1IOHandlerImpl::flush() in ParallelADIO
            S1IOHandler.cpp.o
     118    ld: symbol(s) not found for architecture x86_64
  >> 119    clang: error: linker command failed with exit code 1 (use -v to see
             invocation)
  >> 120    make[2]: *** [lib/libopenPMD.ADIOS1.Parallel.dylib] Error 1
  >> 121    make[1]: *** [CMakeFiles/openPMD.ADIOS1.Parallel.dir/all] Error 2
  >> 122    make: *** [all] Error 2
```

--- a/include/openPMD/IO/IOTask.hpp
+++ b/include/openPMD/IO/IOTask.hpp
@@ -67,7 +67,7 @@ namespace internal
      * The returned strings are compile-time constants, so no worries about
      * pointer validity.
      */
-    std::string operationAsString(Operation);
+    OPENPMDAPI_EXPORT std::string operationAsString(Operation);
 } // namespace internal
 
 struct OPENPMDAPI_EXPORT AbstractParameter

